input/tar2/tar_input.go

```diff
@@ -0,0 +1,163 @@
+package tar2
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha512"
+	"encoding/base64"
+	"hash"
+	"io"
+	"os"
+	"path"
```

Comment:
I think it would have trouble being meaningful.  A file may use both packages.  Also, I'm wary of comments in imports.  There's... talk on the golang-dev list about those becoming a mechanical Thing for dependency management.

So I guess I want... a broad sane way to address the boundary between path/filepath without microcomments, and I was roughly hoping one would evolve along package lines that's suuuper clear and high level. But until that becomes something we can do clearly... local bandage comments are maybe better than nothing -.-